.devcontainer/Caddyfile

```diff
@@ -8,7 +8,7 @@ localhost {
   tls internal
 }
 
-:54321 {
+:8443 {
```

Comment:
`postgres` listens on `15432` in prod so just making this a little more different to prevent confusion.